src/components/ProductDrawer.tsx

```diff
@@ -136,30 +137,62 @@ export const ProductDrawer: FC<ProductDrawerType> = ({
   // };
 
   const navigate = useNavigate();
+  const { logout } = useAuthContext();
+  const handleLogout = () => {
+    logout();
+    navigate("/");
+  };
 
   return (
     <Box sx={{ display: "flex" }}>
       <AppBar position="fixed" open={open}>
-        <Toolbar>
-          <IconButton
-            color="inherit"
-            aria-label="open drawer"
-            onClick={handleDrawerOpen}
-            edge="start"
-            sx={{ mr: 2, ...(open && { display: "none" }) }}
-          >
-            <MenuIcon />
-          </IconButton>
-          <Typography variant="h6" noWrap component="div">
-            Filtry
-          </Typography>
+        <Toolbar
+          sx={{
+            display: "flex", // Enable flexbox
+            justifyContent: "space-between", // Space between left and right items
+            alignItems: "center", // Center items vertically
+          }}
+        >
+          <Box sx={{ display: "flex", alignItems: "center" }}>
+            <IconButton
+              color="inherit"
+              aria-label="open drawer"
+              onClick={handleDrawerOpen}
+              edge="start"
+              sx={{ mr: 2, ...(open && { display: "none" }) }}
+            >
+              <MenuIcon />
+            </IconButton>
+            <Typography variant="h6" noWrap component="div">
+              Filtry
```

Comment:
wszedzie mamy angielski :) 
```suggestion
              Filters
```